mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1154,6 +1156,8 @@ def _get_args_for_metrics(self, extra_metric, eval_df):
         params_not_found = []
         # eval_fn has parameters (eval_df, builtin_metrics) for backwards compatibility
         if len(parameters) == 2:
+            param_0_name, param_1_name = parameters.keys()
+        if len(parameters) == 2 and param_0_name != "predictions" and param_1_name != "targets":
```

Comment:
Fixing because this fails when the eval_fn has parameters (predictions, targets).